docs/retrieve.txt

```diff
@@ -0,0 +1,248 @@
+.. _laravel-retrieve:
+
+==============
+Retrieve Data
+==============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find one, find many, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to retrieve data from your MongoDB
+collections using **find operations**. Find operations are commands that
+retrieve documents from the server.
+
+This guide includes the following sections:
+
+- :ref:`laravel-find-setup`
+- :ref:`laravel-find-all`
+- :ref:`laravel-find-one`
+- :ref:`laravel-modify-find`
+- :ref:`laravel-addtl-info`
+
+.. _laravel-find-setup:
+
+Prerequisites
+-------------
+
+.. TODO: link to Quick Start once it's published
+
+The examples in this guide use the ``movies`` collection in the ``sample_mflix`` database,
+which is included in the MongoDB Atlas sample dataset. For instructions on loading the sample
+dataset into your database, see the Quick Start or the :atlas:`Get Started with Atlas Guide
+</getting-started/#atlas-getting-started>`.
+
+Before running the examples in this guide, you must create the following classes:
+
+- Model class to represent the ``movies`` collection
+- Controller class to store and run the find operations
+- View class to display the results of the find operations
+
+For instructions on creating these classes, see the Quick Start.
+
+.. _laravel-find-all:
+
+Find All Documents
+------------------
+
+To retrieve all documents in a collection, use the ``all()`` method.
```

Comment:
Issue:

I think "use the ___ method" might not be clear as to what you're calling it on.

Suggestion:
I think it could be helpful to make the following changes:
- Describe connection between "MongoDB collection" and "Laravel Model" in the Overview section.
- Provide specifics on what the user should call the "all()" method on.